static/js/app.js

```diff
@@ -0,0 +1,60 @@
+function add_troop() {
+    console.log("create post is working!")
+    var form_data = $('#troop_modal_form').serialize();
+
+    $.ajax({
+        url : "/register/troop/",
+        type : "POST",
+        data : form_data,
+
+        success : function(created_troop_json_string) {
+            var parse = JSON.parse(created_troop_json_string);
+            var created_troop_json = JSON.parse(parse);
+
+            //Add created Council to troop select widget
+            var register_select_widget = $("#id_scout-0-troop");
```

Comment:
How can we change this so we can reuse it on a page with more than one scout (i.e. after we implement the story "allow parents to register multiple scouts simultaneously)?